core/field.js

```diff
@@ -160,6 +160,12 @@ Blockly.Field.prototype.init = function() {
 };
 
 /**
+ * Initializes the model of the field. No-op by default.
```

Comment:
Initializes the model of the field after it has been installed on a block.  No-op by default.